### ISSUE TITLE
checkoutCompleteWithTokenizedPaymentV3 'Field 'payment' doesn't exist on '

### ISSUE THREAD

**Issue by afetter** [imsujan276/shopify_flutter#15]
Hey,

I'm trying to complete the checkout but I'm not able. It is because an error occurs on `checkoutCompleteWithTokenizedPaymentV3`

I'm using version 1.1.0
```

final shopifyStore = ShopifyStore.instance;
      final shopifyAuth = ShopifyAuth.instance;

      final currentUser = await shopifyAuth.signInWithEmailAndPassword(
          email: 'ande****@gmail.com', password: '*****');

      final bestSellingProducts = await shopifyStore.getAllProducts();

      var items = List<LineItem>.empty(growable: true);
      items.add(LineItem(
          quantity: 1,
          variantId: bestSellingProducts[0].productVariants[0].id,
          title: bestSellingProducts[0].title,
          id: bestSellingProducts[0].id));

      var address = Address(
          address1: '11 Hinkler Avenue,
          city: 'Sydney',
          country: 'Australia',
          countryCode: 'AU',
          firstName: 'Anderson',
          lastName: 'Fetter',
          phone: '044444444',
          zip: '2229');

      checkout = await shopifyCheckout.createCheckout(
        lineItems: items,
        mailingAddress: address,
        email: 'ande*******@gmail.com',
      );

      final idempotencyKey = UniqueKey().toString();
      await shopifyCheckout.shippingAddressUpdate(checkout.id, address);
      var completed = await shopifyCheckout
          .checkoutCompleteWithTokenizedPaymentV3(checkout.id,
              checkout: checkout,
              token: r'CQ32pyIRCmIEfekpX8x=',
              paymentTokenType: PaymentTokenType.APPLE_PAY,
              idempotencyKey: idempotencyKey,
              amount: '1.00',
              currencyCode: 'AUD');
```
Error:

_**Field 'payment' doesn't exist on type 'CheckoutCompleteFreePayload' AND
Variable $payment is declared by checkoutCompleteWithTokenizedPaymentV3 but not used**_

Please let me know if I'm doing something wrong, of if you need some help.

**Comment by imsujan276** [comment]
Thank you for the issue. It seems like I have missed applying the payment variable in the mutation. I have updated the package. It should be fixed in v1.1.1

**Comment by afetter** [comment]
@imsujan276 Thank you
